### ISSUE TITLE
stat: Documentation for ExKurtosis is confusing

### ISSUE THREAD

**Issue by romanwerpachowski** [gonum/gonum#1569]
### What are you trying to do?

I'm trying to understand what ExKurtosis is calculating, by reading its docstring.

### What did you do?

Read: https://github.com/gonum/gonum/blob/master/stat/stat.go#L363

### What did you expect to happen?

Understand the implemented function.

### What actually happened?

I read "ExKurtosis returns the population excess kurtosis of the sample" and got confused. Does the function calculate an unbiased estimator of population kurtosis?

### What version of Go and Gonum are you using?

Master branch at HEAD.

### Does this issue reproduce with the current master?

Yes.

**Comment by kortschak** [comment]
Yes, it's a little confusing. The documentation reads as though it's the [biased estimator](https://en.wikipedia.org/w/index.php?title=Kurtosis&oldid=1000787532#A_natural_but_biased_estimator), but the code is the [unbiased estimator](https://en.wikipedia.org/w/index.php?title=Kurtosis&oldid=1000787532#Standard_unbiased_estimator).
